ecs-query.tf

```diff
@@ -35,11 +35,11 @@ resource "aws_ecs_task_definition" "jaeger_query" {
                         "name": "JAEGER_ENDPOINT",
                         "value": "http://${data.aws_lb.jaeger.dns_name}:14268/api/traces"
                     EOS
-                    : <<EOS
+  : <<EOS
                         "name": "JAEGER_DISABLED",
                         "value": "true"
                     EOS
-                    }
+}
```

Comment:
has this lost formatting?

Reply:
i think so because fmt test was only added in this PR